sensors/openstack_router_issue.yaml

```diff
@@ -0,0 +1,33 @@
+---
+class_name: OpenstackRouterSensor
+entry_point: src/openstack_router_sensor.py
+description: Monitor for issues with openstack routers
+poll_interval: 1800 # 30 mins
```

Comment:
Nit: could we do every 10 minutes since this could be quite disruptive when a user does this and 29 minutes to have something call out probably means we've manually tracked it down by that point anyway

AFAIK it's a relatively lightweight check, but if it's heavy we can increase it later on?